about.html

```diff
@@ -0,0 +1,93 @@
+        </div>
+    </section>
+    <section class="menu-a">
+        <div class="nav-menu-a">
+            <div><a href="./index.html"><img src="./images/img-logo.jpg">ikoote PSS</a></div>
+            <div class="nav-a">
+                <a href="./index.html">Home</a>
+                <a href="program">Program</a>
+                <a href="#">Join</a>
+                <a href="#sponsors">Sponsors</a>
+                <a href="#speakers">Feature Speakers</a>
+                <a href=""><button type="button" class="btn">PSS campaign</button></a>
+            </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="header-pag">
+            <img src="./images/mobile-menu-icon-png-4.jpg" class="menu-ba">
+            <nav class="nav-menu" id="nav-menu">
+                <span class="menu-line" id="menu-line">
+                    <img src="./images/Icon-popDD.png" class="menu-2" id="menu-2">
+                    <ul class="menu-br">
+                        <li class="menu-b"><a href="./index.html" class="bb">Home</a></li>
+                        <li class="menu-b"><a href="#program" class="bb">Program</a></li>
+                        <li class="menu-b"><a href="#" class="bb">Join</a></li>
+                        <li class="menu-b"><a href="#sponsers" class="bb">Sponsors</a></li>
+                        <li class="menu-b"><a href="#speakers" class="bb">Feature Speakers</a></li>
+                    </ul>
+                </span>
+            </nav>
+            <div class="hed-2">
+                <h2> "Hello! Galaxy Citizens" </h2>
+                <h1>PHYSICS AND SPACE SCIENCE CONFERENCE 2023</h1>
+            </div>
+            <div class="conf2"><p>Physics and Space Science Conference aims to bring together leading academic scientists, researchers and research scholars to exchange and share their experiences and research results on all aspects of Physics and Space Science Conference.</p></div>
+            <p>Please contact us per Email for any further questions <br> about PSS Galatic Conference 2023</p>
+            <h4>ikootepreim@gmail.com</h4>
+        </div>
+        <div class="logo">
+            <h2>Physics and Space Science Conference 2023 Logo </h2>
+            <img src="./images/img-red.jpg">
+            <p>The logo of PSS Galaxy Conference 2023 was decided through the logo competition in november 2022</p>
+            <div class="logo2"><img src="./images/img-logo.jpg" id="logo"><h4>ikoote PSS</h4></div>
+        </div>
+        <div class="past">
+            <h2>See the past PSS Galaxy Conference</h2>
+            <img src="./images/img-red.jpg">
+            <p>Take a look at the last two PSS Galaxy Conferences which took place in Jinja and Kamuli</p>
+            <div class="psss">
+                <div class="pss1"><h3>2021 <br>  PSS Conference Jinja</h3></div>
+                <div class="pss2"><h3> 2022 <br>  PSS Conference Kamuli</h3></div>
+            </div>
+            
```

Comment:
- [ ] Please update this section for the overlay color to match the original design 

- Original Design 👇 
![image](https://user-images.githubusercontent.com/75126481/217767843-77f2c0f7-d31e-4f7b-ae78-bcce06b66291.png)

- Your design 👇 
![image](https://user-images.githubusercontent.com/75126481/217767991-8468bc69-b4e6-44d8-90ee-dce5956b933c.png)